nginx/nginx.conf

```diff
@@ -23,7 +23,7 @@ http {
   add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload';
 
   # CSP
-  add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' data: https://www.googletagmanager.com https://www.google-analytics.com; img-src 'self' https://www.google-analytics.com; style-src 'self'; font-src 'self'; object-src 'none'; frame-src ##APP_SERVER## https://www.youtube.com/; connect-src ##APP_SERVER## ##HMDA_API_SERVER## https://ffiec-api.cfpb.gov https://www.google-analytics.com;";
+  add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' data: https://www.googletagmanager.com https://www.google-analytics.com; img-src 'self' data: https://www.google-analytics.com; style-src 'self' 'unsafe-inline'; font-src 'self' data:; object-src 'none'; frame-src https://www.youtube.com/; connect-src 'self' https://www.google-analytics.com https://s3.amazonaws.com;";
```

Comment:
Is the s3 domain required here? Looks like we're only loading static documents there via anchor tags.

Reply:
7134383 - which also adds back the caching rules.